client/docker-compose.yml

```diff
@@ -11,10 +11,27 @@ services:
     restart: 'no'
     depends_on:
       - cleaning-scripts
+    networks:
+      - arkhn
 
   cleaning-scripts:
     image: arkhn/cleaning-scripts:latest
     container_name: cleaning-scripts
     env_file: .env.docker
     ports:
-      - 5001:5001
+      - 5000:5000
+    networks:
+      - arkhn
+
+  pagai:
```

Comment:
Do you also need to tell that pyrog_client depends on pagai above?

Reply:
not really but it will be clearer. done.